### PR TITLE
[Chat] Add image placeholder and broken image to my messages

### PR DESCRIPTION
**Pull request by Leah-Xia-Microsoft**
# What

image
image

# Why

# How Tested

# Process & policy checklist

- [ ] I have updated the project documentation to reflect my changes if necessary.
- [ ] I have read the [CONTRIBUTING](https://github.com/Azure/communication-ui-library/blob/main/CONTRIBUTING.md) documentation.

**Is this a breaking change?**

- [ ] This change causes current functionality to break.

### CODE REVIEW

File: packages/react-components/src/components/styles/MessageThread.styles.ts [hunk 1]
```diff
@@ -147,6 +147,46 @@ export const useChatMessageRenderStyles = makeStyles({
   }
 });
 
+const bodyWithPlaceholderImage = (myMessage: boolean = false): GriffelStyle => {
```
Comment (vhuseinova-msft) [inline packages/react-components/src/components/styles/MessageThread.styles.ts:150]: Please add styles to `useChatMyMessageStyles` instead as we shouldn't use `GriffelStyle` directly
The change probably can be part of `body` item

image

Comment (vhuseinova-msft) [reply packages/react-components/src/components/styles/MessageThread.styles.ts:150]: it looks like the separate style was created to share between my message and message styles. Please add the styles to `useChatMessageCommonStyles` instead

File: packages/react-components/src/components/styles/MessageThread.styles.ts [hunk 2]
```diff
@@ -147,6 +147,46 @@ export const useChatMessageRenderStyles = makeStyles({
...
+  return {
+    '& img[src=""]': {
+      display: 'block',
+      position: 'relative',
+      marginBottom: '5px',
+      width: myMessage ? '12rem' : undefined,
+      height: myMessage ? '12rem' : undefined
```
Comment (jpeng-ms) [inline packages/react-components/src/components/styles/MessageThread.styles.ts:157]: why is it for "myMessage" that we hard code the width and height to be 12rem?

Comment (Leah-Xia-Microsoft) [reply packages/react-components/src/components/styles/MessageThread.styles.ts:157]: @jpeng-ms We're planning not to add width and height to the inline image we send out. In that case, the placeholder don't have the real image size to render with. This is not the case for remote messages we receive from Teams, it comes with width and height

**Review by vhuseinova-msft — APPROVED** [review]

File: packages/react-components/src/components/styles/MessageThread.styles.ts [hunk 3]
```diff
@@ -375,6 +347,41 @@ export const useChatMessageCommonStyles = makeStyles({
...
+      display: 'block'
+    },
+    '& .broken-image-wrapper': {
+      width: '12rem',
+      height: '12rem',
+      marginTop: '0.75rem',
+      display: 'flex',
+      justifyContent: 'center',
+      alignItems: 'center',
+      outline: `1px solid ${tokens.colorNeutralForegroundDisabled}`,
```
Comment (vhuseinova-msft) [inline packages/react-components/src/components/styles/MessageThread.styles.ts:374]: This should be ...shorthands.outline:...
https://griffel.js.org/react/api/shorthands/
https://developer.mozilla.org/en-US/docs/Web/CSS/outline

**Review by vhuseinova-msft — APPROVED** [review]

File: packages/react-components/src/components/styles/MessageThread.styles.ts (hunk 2 above)
Comment (jpeng-ms) [reply packages/react-components/src/components/styles/MessageThread.styles.ts]: can you add some comments so we don't forget about the reasons?

File: packages/react-components/src/components/styles/MessageThread.styles.ts [hunk 4]
```diff
@@ -381,6 +355,41 @@ export const useChatMessageCommonStyles = makeStyles({
...
+      backgroundSize: 'center',
+      position: 'absolute',
+      width: '100%',
+      height: '100%',
+      top: '0',
+      left: '0',
+      display: 'block'
+    },
+    '& .broken-image-wrapper': {
+      width: '12rem',
```
Comment (palatter) [inline packages/react-components/src/components/styles/MessageThread.styles.ts:376]: We hardcode this to 12rem in bodyWithPlaceholderImage as well.

Can we create a common const at the top of the file?

**Review by palatter — APPROVED** [review]

**Pull request merged**